docs/erasure/storage-class/README.md

```diff
@@ -1,28 +1,26 @@
 # Minio Storage Class Quickstart Guide [![Slack](https://slack.minio.io/slack?type=svg)](https://slack.minio.io)
 
-Minio server supports storage class in erasure coding mode. This allows configurable data and parity disks per object.
+Minio Server supports two storage classes in erasure coding mode: `Reduced Redundancy` class and `Standard` class. These classes enable Minio Server to save an object across a configurable number of data and parity drives.
 
-## Overview
+These classes are defined by setting environment variables and specifying the data and parity hard drives before starting Minio Server. The storage class to use for an object is set by defining the `x-amz-storage-class` metadata field. 
```

Comment:
IMO we can drop this paragraph, below we have steps on how to set storage classes.